### PR TITLE
TRIB-203: updates database migration scripts with context="test"

### PR DESCRIPTION
**Pull request by mrsbluerose**

### CODE REVIEW

**Comment by mrsbluerose** [comment]
I interpreted this test as adding context="test" to existing migration scripts. In the past, we've added to new scripts to undo old work and redo it however needed. If you prefer I take that second approach, my only concern was v.0.0.1. That script is somewhat involved and sets up a lot of data. I would want to consult about that one and the affects it may have before taking the second approach.

File: src/main/resources/db/migration/changelog-v0.0.1.xml [hunk 1]
```diff
@@ -61,14 +63,16 @@
 				<constraints nullable="false"/>			
 			</column>
 		</createTable>
-		
+	</changeSet>
+
+	<changeSet author="haxwell" id="v0.0.1-4" context="test">
```
Comment (haxwell) [inline src/main/resources/db/migration/changelog-v0.0.1.xml:68]: These INSERTs are production level, not just for testing.

File: src/main/resources/db/migration/changelog-v0.0.5.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
                       http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
-    <changeSet author="maryam" id="v0.0.5-1">
+    <changeSet author="maryam" id="v0.0.5-1" context="test">
```
Comment (haxwell) [inline src/main/resources/db/migration/changelog-v0.0.5.xml:8]: This is production level as well.

**Review by haxwell — APPROVED** [review]

**Comment by haxwell** [comment]
Oops this actually was not ready for approval. The suggestions I made above are actually necessary.

**Review by haxwell — APPROVED** [review]

**Pull request merged**